src-java/kilda-persistence-tinkerpop/src/test/java/org/openkilda/persistence/ferma/repositories/FermaModelUtils.java

```diff
@@ -60,35 +59,14 @@ public static HaFlowPath buildHaFlowPath(
                 .build();
     }
 
-    /**
-     * Builds HaSubFlowEdge object.
-     */
-    public static HaSubFlowEdge buildHaSubFlowEdge(String haFlowId, HaSubFlow haSubFlow, MeterId meterId) {
-        return HaSubFlowEdge.builder()
-                .haFlowId(haFlowId)
-                .haSubFlow(haSubFlow)
-                .meterId(meterId)
-                .build();
-    }
-
-    /**
-     * Builds HaSubFlowEdge objects.
-     */
-    public static Set<HaSubFlowEdge> buildHaSubFlowEdges(
-            String haFLowId, HaSubFlow subFlow1, HaSubFlow subFlow2, MeterId meterId1, MeterId meterId2) {
-        HaSubFlowEdge edge1 = buildHaSubFlowEdge(haFLowId, subFlow1, meterId1);
-        HaSubFlowEdge edge2 = buildHaSubFlowEdge(haFLowId, subFlow2, meterId2);
-        return Sets.newHashSet(edge1, edge2);
-    }
-
     /**
      * Builds HaSubFlow object.
      */
     public static HaSubFlow buildHaSubFlow(
-            String subFlowId, SwitchId switchId, int port, int vlan, int innerVlan, String description) {
+            String subFlowId, Switch sw, int port, int vlan, int innerVlan, String description) {
```

Comment:
no need to shorten, use "switch" instead

Reply:
`switch` is reserved word in Java. We can't use it